packages/composables/src/getters/facetGetters.ts

```diff
@@ -13,6 +13,7 @@ import {
   getCategoryTree as buildCategoryTree,
   getCategoryBreadcrumbs as buildBreadcrumbs
 } from './categoryGetters';
+import {Vendor} from '@vue-storefront/spree-api/server/types/vendor';
```

Comment:
Maybe we could follow the convention that we already have an import these inside ../types, then import it from types here?

Reply:
Also export Vendor in api-client's main types file?